algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,20 @@
+function bfs(value, tree) {
+  queue = [];
+
+  queue.unshift(tree);
+
+  while (queue.length > 0) {
+    curNode = queue.pop();
```

Comment:
Codacy found an issue: ['curNode' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6325959)